H/hidapi/build_tarballs.jl

```diff
@@ -41,11 +44,12 @@ install_license ../hidapi/LICENSE*.txt
 platforms = supported_platforms()
 
 products = [
-    LibraryProduct(["libhidapi", "libhidapi-libusb"], :hidapi)
+    LibraryProduct(["libhidapi", "libhidapi-hidraw","libhidapi-libusb"], :hidapi)
 ]
 
 dependencies = [
-    Dependency("libusb_jll", platforms=filter(p -> Sys.islinux(p) || Sys.isfreebsd(p), platforms))
+    Dependency("libusb_jll", platforms=filter(p -> Sys.isfreebsd(p), platforms))
+    BuildDependency("libusb_jll", platforms=filter(p -> Sys.islinux(p), platforms))
```

Comment:
Why the same dependency twice?

Reply:
The idea was to use libusb backend for freebsd, so that a BuildDependency wouldn't suffice.